apps/site/src/app/admin/applicants/components/ApplicantReviewerIndicator.tsx

```diff
@@ -0,0 +1,58 @@
+import {
+	Box,
+	SpaceBetween,
+	StatusIndicator,
+} from "@cloudscape-design/components";
+
+interface IndicatorContainerProps {
+	reviewerNumber: string;
+	type: string;
+}
+
+const IndicatorContainer = ({
+	reviewerNumber,
+	type,
+}: IndicatorContainerProps) => {
+	return (
+		<>
+			<Box variant="awsui-key-label">Reviewer {reviewerNumber}</Box>
+			{type === "notReviewed" ? (
+				<StatusIndicator type="error">Not Reviewed</StatusIndicator>
```

Comment:
Thought: the Pending or In Progress types might make more sense here than Error.